.eslintignore

```diff
@@ -11,6 +11,7 @@
 /tests/screenshot/*
 /tests/test_runner.js
 /tests/workspace_svg/*
+/tests/typescript/dist/*
```

Comment:
This should not be necessary.

Reply:
When this is not ignored, I get an error like below:
```
5:1  error  Parsing error: 'import' and 'export' may appear only with 'sourceType: module'
```

Basically it's trying to run lint on the JS files that are created in `/tests/typescript/dist`